MultiGoogleTranslate.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+"""
+Use Google Translate to translate your sentence into multiple languages. 
+Visit the following link to check available languages: 
+https://cloud.google.com/translate/docs/languages
+To add or remove languages use modifier key when trigger is activated or go to: 
+'~/.config/albert/org.albert.extension.translate/config.json'
+Add or remove elements based on the ISO-Codes that you found on the google documentation page.
+"""
+
+from albertv0 import *
+import json
+import urllib.request
+import urllib.parse
+import os
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Multi Google Translate"
+__version__ = "1.0"
+__trigger__ = "mtr"
```

Comment:
please append a space to the extensions trigger to avoid collisions. the extension consumes arguments anyway